vocabularies/UI.xml

```diff
@@ -1609,6 +1600,17 @@ This is in contrast to [DataFieldForIntentBasedNavigation](#DataFieldForIntentBa
         <Annotation Term="Core.Description" String="The contents of this property must not be propagated to the app-to-app navigation context" />
       </Term>
 
+      <Term Name="Timezone" Type="Edm.String" AppliesTo="EntityContainer Property Parameter">
+        <Annotation Term="Core.Description" String="The point in time represented by the annotated property or all points in time in this entity container shall be presented in the context of the given time zone"/>
+        <Annotation Term="Core.LongDescription">
+          <String>The value list for time zones is not prescribed by this vocabulary.
+            If a [IANA](https://www.iana.org/time-zones) time zone is specified, browsers can perform the necessary conversion.
+            Otherwise, the `Edm.DateTimeOffset` in the annotated property SHOULD contain a [timezoneOffset](https://www.w3.org/TR/2012/REC-xmlschema11-2-20120405/#vp-dt-timezone) that corresponds to the time zone.

```

Comment:
I would remove this sentence unless we want to require that UI5 can do this.

Reply:
Which of the three sentences?